source/_UIAHandler.py

```diff
@@ -189,6 +189,8 @@
 	#UIA_ToolTipClosedEventId:"hide",
 	UIA_Window_WindowOpenedEventId:"UIA_window_windowOpen",
 	UIA_SystemAlertEventId:"UIA_systemAlert",
+	UIA_Drag_DragCompleteEventId: "UIA_dragComplete",  # NOQA: F405

```

Comment:
Please avoid noqa

```suggestion
	UIA.UIA_Drag_DragCompleteEventId: "UIA_dragComplete",
```

Reply:
NOQA is required because without that, local lint test fails- Flake8 doesn't know where these UIA event constants are coming from since we import everything from UIA client support library (COM interfaces).

Reply:
Note that in my suggestion, I suggested to use the UIA import. That should avoid the Flake8 error.